extract_otp_secret_keys.py

```diff
@@ -97,4 +102,9 @@ def print_qr(data):
         otp_url = 'otpauth://{}/{}?'.format('totp' if otp.type == 2 else 'hotp', quote(otp.name)) + urlencode(url_params)
         if args.qr:
             if verbose: print(otp_url)
-            print_qr(otp_url)
+            if not(os.path.exists("qr")):os.mkdir("qr")
```

Comment:
Code formatting:
I suggest either a space after : or put it on a new line

Reply:
Done